From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: GideonWhite1029 <gideonwhite1029@gmail.com>
Date: Fri, 11 Apr 2025 10:35:11 +0300
Subject: [PATCH] Rebrand


diff --git a/src/main/java/io/papermc/paper/ServerBuildInfo.java b/src/main/java/io/papermc/paper/ServerBuildInfo.java
index fb1fe2651e53a9bf46b3632c638e13eea9dcda93..bb1f2a8bf1cdb4f09aad70e4ebcee19979bc0907 100644
--- a/src/main/java/io/papermc/paper/ServerBuildInfo.java
+++ b/src/main/java/io/papermc/paper/ServerBuildInfo.java
@@ -25,6 +25,12 @@ public interface ServerBuildInfo {
      */
     Key BRAND_PURPUR_ID = Key.key("purpurmc", "purpur");
     // Purpur end
+    // PermaSpect start
+    /**
+     * The brand id for PermaSpect.
+     */
+    Key BRAND_PERMASPECT_ID = Key.key("permaspect", "permaspect");
+    // PermaSpect end
     /**
      * Gets the {@code ServerBuildInfo}.
      *
diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
index 69c0ca1b0c0cdbc911ba3f053fafc5624539917d..0ef0d97c6783ad7bd0f2ac2b13e44ead625983ab 100644
--- a/src/main/java/org/bukkit/Server.java
+++ b/src/main/java/org/bukkit/Server.java
@@ -2399,6 +2399,13 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
         }
         // Purpur end
 
+        // PermaSpect start
+        @NotNull
+        public org.bukkit.configuration.file.YamlConfiguration getPermaSpectConfig() {
+            throw new UnsupportedOperationException("Not supported yet.");
+        }
+        // PermaSpect end
+
         /**
          * Sends the component to the player
          *
